packages/api-client/index.ts

```diff
@@ -30,14 +31,21 @@ class APIClient {
     },
   };
   taStatus = {
-    update: async (courseId: number): Promise<TAUpdateStatusResponse> => {
+    checkIn: async (
+      courseId: number,
+      room: string
+    ): Promise<TAUpdateStatusResponse> => {
       const queue = (
-        await this.axios.patch(`/api/v1/courses/${courseId}/ta/change_status`)
+        await this.axios.post(`/api/v1/courses/${courseId}/ta_location/${room}`)
```

Comment:
It might make sense here to put the room in the body of the request here. When I tested it out I got a request url like `/api/v1/courses/1/ta_location/RY%20154`

Reply:
dajin and I were discussing it and I dont' remember why we decided to leave it in the URL but there was a reason. I think it was just cause Nest auto encode/decodes the URLs sent and doing it in a body at that point seemed to be more squeeze for the same quality of juice

Reply:
url encoding is not a bad thing typically. It's not super important where it goes, however, `DELETE` http requests aren't supposed to put stuff in the request body, so it'd be better to put it in the path for when we check out via `DELETE`